index.js

```diff
@@ -198,6 +198,8 @@ async function smartContractInitialization() {
     initialUrls.push('{"ws": "ws://10.200.10.1:30302", "http": "http://10.200.10.1:30302"}')
     initialNodes.push('0xf2C195bE194a2C91e93Eacb1d6d55a00552a85E2')
     initialUrls.push('{"ws": "ws://10.200.10.1:30303", "http": "http://10.200.10.1:30303"}')
+    initialNodes.push('0xde1112f631486CfC759A50196853011528bC5FA0')
```

Comment:
is this added node related to the ERC677 token change?

Reply:
The goal was to deploy another `NodeRegistry`, this shouldn't be here at all, needs to be removed

Reply:
The goal in ETH-51 was to deploy another `NodeRegistry`, this shouldn't be here at all, needs to be removed